hack/.ci/prepare_release

```diff
@@ -16,7 +16,7 @@
 
 set -e
 
-repo_root_dir="$1"
+repo_root_dir="$(realpath $1)"
```

Comment:
I think that the `cp` failure that you described occurs only when prepare_release script is invoked from relative path:

```
# ./.ci/prepare_release

cp: cannot stat './.ci/..': No such file or directory
```

but 

```
# /mnt/.ci/prepare_release
```

succeeds.

In general, it makes sense to support both. 👍